README.md

```diff
@@ -1,4 +1,10 @@
-lesson02
+loacker lesson 02
 ========
 
 web course excercise no 2
+We will practice loacker here
```

Comment:
I'm not sure what `practice loacker` means - please clarify

Reply:
Done